distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc

```diff
@@ -54,8 +54,10 @@ Configure the Security STS LDAP Claims Handler from the ${admin-console}:
 
 |===
 
-.Security STS LDAP Claims Handler Imported Services
+===== Implementation Details
```

Comment:
Having these as section titles will create automatic anchors to this section and the name `Implementation Details` will be too general considering all the other sections that _could_ be named the same. 

I'm not sure we even need to include this information, especially as nothing is imported or exported. Unless we have a use case, I'd advise just deleting these everywhere.